src/modules/tickets/components/TicketEditForm.tsx

```diff
@@ -47,32 +45,31 @@ export default class TicketEditForm extends React.Component<Props, State> {
     };
   }
 
-  onChangeField = <T extends keyof State>(name: T, value: State[T]) => {
-    this.setState({ [name]: value } as Pick<State, keyof State>);
-  };
-
-  renderSidebarFields = () => {
+  renderSidebarFields = callback => {
```

Comment:
Why are you naming saveItem as callback ?
Same in Task
Please name your variables properly

Reply:
Ok, name changed.